x/emissions/keeper/inference_synthesis/synth_palette_forecast_implied_test.go

```diff
@@ -405,7 +405,7 @@ func (s *InferenceSynthesisTestSuite) TestCalcForcastImpliedInferencesEpoch3() {
 			alloraMath.InDelta(
 				expectedValue.Value,
 				actualValue.Value,
-				alloraMath.MustNewDecFromString("0.001"),
+				alloraMath.MustNewDecFromString("0.01"),
```

Comment:
prob we should add the updated version of the simulator outputs instead of lowering precision here, if it's available ofc

Reply:
the updated simulator outputs are included